File: Sorting.java
import java.util.*;
class Sorting{
    public static void main(String args[]){
        int arr[]={8,6,4,7,9,6,5},i;
        Scanner sc = new Scanner(System.in);
        System.out.println("before sorting");
        for(i=0;i<arr.length;i++){
            System.out.print(arr[i]+" ");
        }
        System.out.println();
        for(i=0;i<arr.length-1;i++){
            for(int j=0;j<arr.length-i-1 ;j++){
                if(arr[j]>arr[j+1]){
                   int temp =arr[j];
                   arr[j]=arr[j+1];
                   arr[j+1]=temp;
                }
            }
        }
        System.out.println("After sorting");
        for(i=0;i<arr.length;i++){
            System.out.print(arr[i]+" ");
        }
       
    }
}

File: Sorting.c
class Sorting{
    public static void main(String args[]){
        int arr[7],i;
        Scanner sc = new Scanner(System.in);
        for(i=0;i<7;i++){
            arr[i]=sc.nextInt();
        }
        System.out.println("before sorting");
        for(i=0;i<7;i++){
            System.out.print(arr[i]+" ");
        }
        for(i=0;i<=7-i;i++){
            if(arr[i]>arr[i+1]){
                int temp =arr[i];
                arr[i]=arr[i+1];
                arr[i+1]=temp; 
            }
        }
        System.out.println("After sorting");
        for(i=0;i<7;i++){
            System.out.print(arr[i]+" ");
        }
       
    }
}